people_test.go

```diff
@@ -154,6 +154,8 @@ func TestFindPeople(t *testing.T) {
 	}
 	require.Equal(t, expected, people)
 	require.Equal(t, map[string]int{"alice": 1, "admin": 1, "bob": 4}, nameFreqs)
+	require.Equal(t, map[string]int{"bob@google.com": 3, "alice@google.com": 1, "bad-email@domen": 1,
+		"someone@google.com": 1}, emailFreqs)
```

Comment:
We miss the test for the primary emails assignment. I guess, for the primary names, too.

Reply:
Yes, I was wondering about that. Actually it's kinda tested here in the `FindPeople` function, but separate ones still would better be implemented, yes?

Reply:
Yes, we look at the frequencies here, not at the final result.

Reply:
Ok, will add them